### PR TITLE
[WIP] Testing with the autodll plugin.

### PR DESCRIPTION
**Pull request by arunoda**
Trying to integrate with this [promising plugin](https://github.com/asfktz/autodll-webpack-plugin). 

Still this is a test.

### CODE REVIEW

**Comment by timneutkens** [comment]
Wow those benchmarks look promising 😲

**Comment by nelix** [comment]
Is their anything I can do to help this get on track? While hacking autodll into nextjs via the config is not impossible its a ton of config and even then its not perfect due to some of the other plugins (common chunks).

File: server/build/webpack.js [hunk 1]
```diff
@@ -85,6 +86,18 @@ export default async function createCompiler (dir, { dev = false, quiet = false,
         }
       }
     }),
+    new AutoDllPlugin({
+      inject: true,
+      context: console.dir(),
```
Comment (chuyik) [inline server/build/webpack.js]: Why not use `dir` variable?

**Comment by timneutkens** [comment]
Going to close this, since vendor bundles work different in webpack 4

**Pull request closed without merging**